CONTRIBUTING.md

```diff
@@ -195,6 +195,8 @@ This has the following keys:
 * `platforms` (default: `["linux"]`): A list of supported OSes.
   Can contain `win32`, `linux`, and `darwin` values.
   If not specified, only `linux` is considered supported.
+  Only add extra OSes to the test
+  if there is platform-specific code in a package.
```

Comment:
Just a suggestion: I'd specify platform-specific stubs. PyAutoGUI is a good example that has platform-specific code, but the public module (the one we stub), has platform-agnostic typing. (hence it doesn't need to be stub-tested on multiple platforms).

vobject, cffi, setuptools, colorama, etc. have specific allowlist entries because of windows, these are example of stubs that would benefit from running on multiple OSes. (less exclusions, more completion)

Reply:
Maybe something around the lines of `if there are platform-specific [type hints/typing] in a package's stub`
or `if a stub contains platform-specific types`

Reply:
cc @AlexWaygood as some who is great with wording :)

Reply:
Maybe `if there is platform-specific code in a stubs package` or `if there are platform-specific branches in a stubs package`?